gematria/testing/testdata/test.csv

```diff
@@ -0,0 +1,154 @@
+4883c2014883fa40,0.00001339
```

Comment:
It doesn't seem like this test data is wired up into a test anywhere? Not sure what the best way to wire it up would be, but it would be good to have some test coverage for this.

Reply:
Yeah... I randomly copied pasted from [one bhive benchmark](https://github.com/GranLte/bhive/blob/master/benchmark/sources/sqlite.csv) for testing...